File: storage/alt_interpreters/VisualStudio_IDE/JsonPL/JsonPL/com/middlemind/JsonPL/ProcessUrlSetRunner.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace com.middlemind.JsonPL
{
    public class ProcessUrlSetRunner
    {
        private JsonPlState main;
        private string url;

        public ProcessUrlSetRunner(JsonPlState Main, string Url)
        {
            main = Main;
            url = Url;
        }

        public void run()
        {
            main.processUrlSetExec(url, true);
        }
    }
}

File: storage/alt_interpreters/NetBeans_IDE/JsonPL/src/com/middlemind/JsonPL/ProcessUrlSetRunner.java
package com.middlemind.JsonPL;

/**
 *
 * @author brusc
 */
public class ProcessUrlSetRunner implements Runnable {
    private final JsonPlState main;
    private final String url;
    
    public ProcessUrlSetRunner(JsonPlState Main, String Url) {
        main = Main;
        url = Url;
    }
    
    @Override
    public void run() {
        main.processUrlSetExec(url, true);
    }    
}
